src/signals_notebook/entities/chemical_drawing.py

```diff
@@ -129,3 +240,39 @@ def dump_templates(cls, base_path: str, fs_handler: FSHandler) -> None:
                 template.dump(fs_handler.join_path(base_path, 'templates', entity_type), fs_handler)
         except TypeError:
             pass
+
+    def dump(self, base_path: str, fs_handler: FSHandler) -> None:
+        """Dump ChemicalDrawing
+
+        Args:
+            base_path: content path where create dump
+            fs_handler: FSHandler
+
+        Returns:
+
+        """
+        content = self.get_content()
+
+        if content.content != b'<CDXML />':
+            metadata = {
+                'file_name': content.name,
+                'content_type': content.content_type,
+                **{k: v for k, v in self.dict().items() if k in ('name', 'description', 'eid')},
+            }
+            if content.content_type == self.ContentType.CDXML.value:
```

Comment:
why do you get `value`?

Reply:
I had the same problem in one of the PRs. Python compares data and if we have self.ContentType.CDXML, it will give <ContentType 'cdxml'> for comparison. maybe smth was changed. But here we have 2 possible ways to solve this: 1. use .value 2. override str, repr methods
I see only this meaning of usage

Reply:
Or stop... get_content returns ContentType.CDXML, not value
Maybe in this case we can use without .value

Reply:
it works with value)